docs/src/main/asciidoc/chapters/administration.txt

```diff
@@ -1175,24 +1175,16 @@ the system to full capacity without administrator interaction after many of the
 common failure modes.
 
 ==== Tested Versions
-Another large consideration for Accumulo stability is to use versions of
-software that have been tested together in a VM environment. Any cluster of
-processes that have not been tested together are likely to expose running
-conditions that vary from the environments individually tested in the various
-components. For example, Accumulo's use of HDFS includes many short block
-reads, which differs from the more common full file read used in most
-map/reduce applications. We have found that certain versions of Accumulo and
-Hadoop will include stability bugs that greatly affect overall stability. In
-our testing, Accumulo 1.6.2, Hadoop 2.6.0, and Zookeeper 3.4.6 resulted in a
-stable VM clusters that did not fail a month of testing, while Accumulo 1.6.1,
-Hadoop 2.5.1, and Zookeeper 3.4.5 had a mean time between failure of less than
-a week under heavy ingest and query load. We expect that results will vary with
-other configurations, and you should choose your software versions with that in
-mind.
-
-
-
-
-
-
-
+Each release of Accumulo is built with a specific version of Apache
+Hadoop, Apache ZooKeeper and Apache Thrift.  We expect Accumulo to
+work with versions that are API compatable with those versions.
+However this compatibility is not guaranteed because Hadoop, ZooKeeper
+and Thift may not provide guarantees between their own versions. We
+have also found that certain versions of Accumulo and Hadoop included
+bugs that greatly affected overall stability.  Thrift is particularly
+prone to compatablity changes between versions and it is strongly
+recommended that you use the same version your Accumulo is built
```

Comment:
Would suggest "you must use the same version" instead of "it is strongly recommended". There is no universe (presently) in which you can use a version of Thrift that isn't the one we use. Right?